week-4/InClass/C-more-complex-objects/exercise-2.js

```diff
@@ -8,6 +8,7 @@
 let house = {
     address: "1 Kinning Park",
     previousOwners: ["Claire M.", "John A."],
+    isForSale = false,

```

Comment:
This works, but can you achieve the same thing by modifying the object _after_ it was written down (on line 30, like you did with `previousOwners`), rather than by changing how we defined the object in the first place?